src/framapp-polymer-app/login.html

```diff
@@ -0,0 +1,32 @@
+<link rel="import" href="../../bower_components/polymerfire/firebase-app">
+<link rel="import" href="../../bower_components/polymerfire/firebase-auth">
+
+<dom-module id="login-component">
+    <template>
+      <style>
+
+      </style>
+      <firebase-app
+        auth-domain="project-5613440220430148247.firebaseapp.com"
+        database-url="https://project-5613440220430148247.firebaseio.com"
+        api-key="AIzaSyBKP4cOP508h0JLKmjFvzJooO0MqV8l4fU">
+      </firebase-app>
+      <firebase-auth id="auth" user="{{user}}" provider="google" on-error="handleError">
+      </firebase-auth>
+      <button on-tap="handleClick">Login</button>
+    </template>
+    <script>
+        Polymer({
+            is: 'login-component',
+            properties:{},
+            handleClick: function(){
+              //https://github.com/firebase/polymerfire/blob/master/firebase-auth.html
+              this.$.auth.signInWithEmailAndPassword("framapp@pgs-soft.com", "password")
```

Comment:
when I call this method (based on https://github.com/firebase/polymerfire#firebase-auth ) I get "is not a function" error, and if I check with googl edev tools what is in "this.$.auth" there is no such function (and based on docs it should be...)